src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -355,8 +357,19 @@
   background-clip: padding-box;
   box-shadow: var(--pf-c-dropdown__menu--BoxShadow);
 
-  &.pf-m-align-right {
-    right: 0;
+  @each $breakpoint, $breakpoint-value in $pf-c-dropdown--breakpoint-map {
+    $breakpoint-name: if($breakpoint != "base", -on-#{$breakpoint}, "");
+
+    @include pf-apply-breakpoint($breakpoint, $pf-c-dropdown--breakpoint-map) {
+      &.pf-m-align-right#{$breakpoint-name} {
+        right: 0;
+      }
+
+      // unset alignment set for .pf-m-align-right
+      &.pf-m-align-left#{$breakpoint-name} {
+        right: unset;
```

Comment:
`unset` works here. Though for what it's worth, `unset` needs to check to see if the property is inheritable (`right` is not inheritable), then the value will be `inherit` if it is, or `initial` if it isn't.

So you could set this to `initial` and save the browser that check. I'm assuming it's an extra check anyways...

But the `initial` value for `right` is `auto`, so you could just set this to `auto`, too.